src/displayapp/screens/Music.cpp

```diff
@@ -212,33 +212,30 @@ bool Music::Refresh() {
       UpdateLength();
     }
   } else {
-    lv_label_set_text(txtPlayPause, Symbols::play);
+    lv_label_set_text_static(txtPlayPause, Symbols::play);
   }
 
   return running;
 }
 
 void Music::UpdateLength() {
   if (totalLength > (99 * 60 * 60)) {
-    lv_label_set_text(txtTrackDuration, "Inf/Inf");
+    lv_label_set_text_static(txtTrackDuration, "Inf/Inf");
   } else if (totalLength > (99 * 60)) {
-    char timer[12];
-    sprintf(timer,
-            "%02d:%02d/%02d:%02d",
-            (currentLength / (60 * 60)) % 100,
-            ((currentLength % (60 * 60)) / 60) % 100,
-            (totalLength / (60 * 60)) % 100,
-            ((totalLength % (60 * 60)) / 60) % 100);
-    lv_label_set_text(txtTrackDuration, timer);
+    lv_label_set_text_fmt(txtTrackDuration,
+                          "%02d:%02d/%02d:%02d",
+                          (currentLength / (60 * 60)) % 100,
+                          ((currentLength % (60 * 60)) / 60) % 100,
+                          (totalLength / (60 * 60)) % 100,
+                          ((totalLength % (60 * 60)) / 60) % 100);
   } else {
     char timer[12];
```

Comment:
this is probably also obsolete

Reply:
Yes, thanks for pointing that out. I'll remove it too.